src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -314,10 +332,11 @@ public override Task SendUsersAsync(IReadOnlyList<string> userIds, string method
             return Task.CompletedTask;
         }
 
-        private Task PublishAsync(string channel, byte[] payload)
+        private async Task PublishAsync(string channel, byte[] payload)
         {
+            await EnsureRedisServerConnection();
```

Comment:
This shouldn't be needed right?

Reply:
This is needed since there are scenarios where OnConnected isn't called

Reply:
`IHubContext` could be used before any connections actually exist on this server. @mikaelm12 's right here, we aren't guaranteed that the connection is up when we get here.